cluster/saltbase/salt/flannel/init.sls

```diff
@@ -0,0 +1,42 @@
+flannel-tar:
+  archive:
+    - extracted
+    - user: root
+    - name: /usr/local/src
+    - makedirs: True
+    - source: https://github.com/coreos/flannel/releases/download/v0.5.5/flannel-0.5.5-linux-amd64.tar.gz
```

Comment:
OK.  Can you throw a TODO here or somewhere to convert this to a static pod once we have it all running as a plugin?